initfiles/etc/DIR_NAME/CMakeLists.txt

```diff
@@ -15,6 +15,72 @@
 #    limitations under the License.
 ################################################################################
 
+
+if(WSSQL_SERVICE)
```

Comment:
if !WSSQL_SERVICE, what do $WSSQL_BINDING_ENV, and $WSSQL_SERVICE_ENV resolve to? blank line?

Reply:
Also, I'm wondering if this WsSQL specific binding and service block should live in the esp/services/ws_sql/CMakeList.txt... 
Are there timing/sequence restrictions that force you to store these two blocks here?

Reply:
I think given how the environment.xml file gets generated, those variables need to be declared before the configure_file is hit.  Which I can't guarantee if it's in esp/services/ws_sql/CMakeLists.txt.  But I can try and figure out a way to do it, because I do agree that it makes more sense for those two variables to be declared in that file.

Reply:
Yes those variables will be undeclared and just a blank.